index.html

```diff
@@ -12110,13 +12122,21 @@ <h2>Context Definitions</h2>
   <p class="changed">If an <a>expanded term definition</a> has an <code>@context</code> <a>entry</a>,
     it MUST be a valid <code>context definition</code>.</p>
 
+  <p class="changed">If a <a>context</a> contains the <code>@import</code>
+    <a>keyword</a>, its value MUST be an <a>absolute IRI</a> or <a>relative IRI</a>.
+    When used as a reference from an `@import`, the referenced <a>context definition</a> MUST NOT
+    include an `@import` key, itself.</p>
+
   <p class="changed">If the <a>expanded term definition</a> contains the <code>@nest</code>
     <a>keyword</a>, its value MUST be either <code>@nest</code>, or a term
     which expands to <code>@nest</code>.</p>
 
   <p class="changed">If the <a>expanded term definition</a> contains the <code>@prefix</code>
     <a>keyword</a>, its value MUST be <code>true</code> or <code>false</code>.</p>
 
+  <p class="changed">If the <a>expanded term definition</a> contains the <code>@propagate</code>
```

Comment:
cf. my comment above, I don't think that an expanded term definition can contain the `@propagate` keyword...